### DISCUSSION TITLE
Is there a way to x/y plot different upscalers?

### DISCUSSION THREAD

**Discussion by Vektor369** [Coyote-A/ultimate-upscale-for-automatic1111#103] [Q&A]
Hi all.

Just what the title says. Is there a way to plot the different upscalers at all? They all seem to come out with results that are better at doing different parts of my images. For example, one will have a good face, while another will have good hair, and another will have a good background. I'm just manually having to run them all, and it gets tiresome basically babysitting them all. My machine isn't the greatest, so it takes me upwards of nine minutes to render each one. It would be so much better if I could just set up the ones I want, and then go eat dinner or something.

Thanks in advance!

**Comment by technerd-official** [comment]
Maybe something like agent scheduler would help automate your workflow. Have to test it out myself.
https://github.com/ArtVentureX/sd-webui-agent-scheduler
